### ISSUE TITLE
[Navigate to Main Body content of "Adaptive cards" tab]: Voice Access is not able to click 'Privacy & Cookies' and 'Storybook.js' from the footer section of Adaptive Cards

### ISSUE THREAD

**Issue by Shireen2000** [microsoft/AdaptiveCards#7506]
### Target Platforms

Other

### SDK Version

103.0.1255.0 

### Application Name

Adaptive Cards

### Problem Description

https://microsoft.visualstudio.com/OS/_queries/edit/39920440/?triage=true

Test Environment:
OS: 22H2 (OS Build 25120.1000)
Browser: Edge Dev
Browser Version: Version 103.0.1255.0 (Official build) dev (64-bit)
URL: [Adaptive Cards](https://adaptivecards.io/)
Tool: Voice Access

Repro Steps:
Launch the above URL.
Command: "Voice access wake up"
Navigate to the footer section of the website.
Command: 'Click Privacy & Cookies' or 'Click Storybook.js' from 'Footer Section'
Observe the issue.
Actual Result:
Voice Access is not able to click 'Privacy & Cookies' and 'Storybook.js' from the footer section of Adaptive Cards.
However, we are able to click 'Privacy & Cookies' and 'Storybook.js' links using Grid view.

Expected Result:
Voice Access users should be able to click 'Privacy & Cookies' and 'Storybook.js' from the footer section of Adaptive Cards.

Observation: 'IsKeyboardFocusable' property is set as 'True' for both 'Privacy & Cookies' and'Storybook.js'.

User Experience:

People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access to use OneNote may not be able edit or type they will find difficulty in accessing 'Canvas area' and 'Canvas Title' edit field. Voice access users will get impact and will not able to interact with controls with pre-defined Voice access commands.
 
MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)
 
Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

To learn more about Voice Access and how an application can support its different modes, Refer: [VoiceAccess Testing - OSGWiki](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.osgwiki.com%2fwiki%2fVoiceAccess_Testing&data=05%7c01%7cv-shiwasi%40microsoft.com%7c00297be07a15467b634408da325a247c%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637877659989570305%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=sQj%2Bb4UzcB2jUBsX3r%2f4%2BjkjzUrXncXKGhGCIuu8vc4%3D&reserved=0)

### Card JSON

```JSON
NA
```

**Comment by Shireen2000** [comment]
https://user-images.githubusercontent.com/96283152/173514607-a47bd663-f434-4f4d-851c-a8dc022663d8.mp4
![MAS4 3 1 - With VA not able to click Privacy and Cookes](https://user-images.githubusercontent.com/96283152/173514655-a41f5a5e-43d0-4e25-9a30-0f5cd46da465.png)
![MAS4 3 1 - With VA not able to click Privacy and Cookies Cde](https://user-images.githubusercontent.com/96283152/173514698-277f43c5-a326-43a0-90bc-cdc3f38f403f.png)

**Comment by anna-dingler** [comment]
Hi @Shireen2000! I believe this should be a bug on the Voice Access team because it should be able to recognize special characters. 

The Voice Access documentation also recommends using a partial name when the name contains a special character (see [this documentation](https://support.microsoft.com/en-us/topic/use-voice-to-interact-with-items-on-the-screen-e1b53c8a-7765-495a-9611-d9b37c008319)). Following this recommendation, both items can be clicked using Voice Access.

For `Privacy & Cookies` specifically, we pull this content from the Universal Header and Footer service. The most recent update changed `Privacy & Cookies` to only say `Privacy`, so this is a non-issue altogether.

Let me know if you have any questions!
